src/dlangui/dialogs/filedlg.d

```diff
@@ -597,9 +598,9 @@ class FileDialog : Dialog, CustomGridCellAdapter {
         _fileList.layoutWidth(FILL_PARENT).layoutHeight(FILL_PARENT);
         _fileList.resize(4, 3);
         _fileList.setColTitle(0, " "d);
-        _fileList.setColTitle(1, "COL_NAME"d);
-        _fileList.setColTitle(2, "COL_SIZE"d);
-        _fileList.setColTitle(3, "COL_MODIFIED"d);
+        _fileList.setColTitle(1, dtext("COL_NAME"c));
```

Comment:
I think currently this lines are unable to translate. This need a change in `StringGridWidget`. Correct me if I'm wrong. Why you use `dtext()`?